### PR TITLE
New Tutorial: Coral Talk on Cloud Run

### PR DESCRIPTION
**Pull request by vyolla**

### CODE REVIEW

File: tutorials/coral-talk-on-cloud-run/index.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+tags: cloud-run, memorystore
+date_published: 2022-04-04
+---
+
+Bruno Patrocinio | Customer Engineer | Google
+
+
+<p style="background-color:#CAFACA;"><i>Contributed by Google employees.</i></p>
+
+This tutorial describes how to deploy open-source commeting platform, [Coral Talk](https://docs.coralproject.net/)  on Google Cloud Platform using managed services.
```
Comment (iennae) [inline tutorials/coral-talk-on-cloud-run/index.md:14]: This tutorial describes how to deploy an open-source commenting platform,

File: tutorials/coral-talk-on-cloud-run/index.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  - [Cloud Run](https://cloud.google.com/run/docs)
+  - [Container Registry](https://cloud.google.com/container-registry/docs)
+  - [Memorystore](https://cloud.google.com/memorystore/docs)
+  - [Compute Engine](https://cloud.google.com/compute/docs)
+  - [`gcloud`](https://cloud.google.com/sdk/docs)
+  - [Docker](https://docs.docker.com/engine/reference/commandline/run)
+
+## Objectives
+
+* Learn how to create and deploy services using `gcloud` commands.
```
Comment (iennae) [inline tutorials/coral-talk-on-cloud-run/index.md:34]: is this the specific objective for this tutorial? it feels like it's showing an example of an application deployment with Cloud Run and Memory Store?

File: tutorials/coral-talk-on-cloud-run/index.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+3. Enable the Cloud Run and Container Registry APIs. For details, see [ENABLING APIs](https://cloud.google.com/apis/docs/getting-started#enabling_apis).
+   
+4. Add role *Container Registry Service Agent* in service account `[project-id]-compute@developer.gserviceaccount.com`
+
+
+## Detailed steps
+
+
+### Download images and upload to Google Container Registry
```
Comment (iennae) [inline tutorials/coral-talk-on-cloud-run/index.md:73]: Recommend using Artifact Registry instead unless there is some kind of limitation or requirement to use Container Registry.

File: tutorials/coral-talk-on-cloud-run/index.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+--platform=managed \
+--region=us-central1 \
+--project=my-project
+```
+
+- Add environment variables `MONGODB_URI`, `REDIS_URI` and `SIGNING_SECRET`
+- Add VPC Connector
+  
+### Access service url to config Coral Talkl
+And all done :D
```
Comment (iennae) [inline tutorials/coral-talk-on-cloud-run/index.md:141]: Suggest adding a short sentence reiterating what just happened. 

something like .. 

You've successfully deployed the Mongo and Coral Talk docker containers to Registry, configured your serverless instances to connect directly to your Virtual Private Cloud network, configured a Memorystore Redist instance,  and set up a VM using the Mongo container, and deployed Coral Talk Service to Cloud Run.

**Review by iennae — APPROVED** [review]
LGTM

**Review by zezo2019 — APPROVED** [review]

**Pull request merged**